docs/source/markdown/rhc-configure.8.md

```diff
@@ -0,0 +1,181 @@
+% rhc-configure 8
+
+# NAME
+
+rhc-configure - Configure RHC features before or after registration
+
+# SYNOPSIS
+
+```
+rhc configure features show
+rhc configure features enable FEATURE
+rhc configure features disable FEATURE
+```
+
+# DESCRIPTION
+
+The **rhc configure** command allows users to manage feature levels and individual features on a host, both before and after registration with Red Hat services. This provides a user-friendly abstraction for changing the host's connectivity level without requiring system re-registration.
+
+Features can be managed independently, allowing users to enable or disable specific functionality such as content access, analytics data collection, and remote management capabilities.
+
+# FEATURES
+
+The following features are available for configuration:
+
+**content**
+: Access to package repositories. This is the base feature that provides system identity with Red Hat and enables content access.
+
+**analytics**
+: Red Hat Lightspeed data collection. Enables insights-client for system analytics and recommendations. Depends on the content feature.
+
+**remote-management**
+: Red Hat Lightspeed remote management capabilities. Enables yggdrasil service for remote system management. Depends on the analytics feature.
+
+# SUBCOMMANDS
+
+**show**
+: List all available features with their current configuration state and runtime state. The output displays a table with columns for FEATURE, CONFIG (user preference), STATE (actual system state), and DESCRIPTION.
+
+**enable** FEATURE
+: Enable the specified feature. This sets the user preference and, if the system is already registered, activates the feature on the system. Dependencies are automatically enabled.
```

Comment:
Suggestion: I would not use the term the "registered". I would use term the "connected", because we have command `rhc connect`.